docs/backend/data-update-scripts.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Data Update Scripts
+---
+
+## What are Data Update Scripts?
+
+Data Update Scripts were introduced recently in
+[this PR](https://github.com/thepracticaldev/dev.to/pull/6025) and allow us to
+run any data updates we might need. For example, if we added a column to the
+database and then wanted to backfill that column with data, rather than going
+and manually doing it in a console, we would use a DataUpdateScript. Another
+example might be adding a new attribute to Elasticsearch. We could then use a
+DataUpdateScript to reindex all of our models.
+
+## How it works
+
+First off, we added a
+[DataUpdateScript model](https://github.com/thepracticaldev/dev.to/blob/master/app/models/data_update_script.rb)
+to Rails and a corresponding database table. This table is what we use to keep
+track of what scripts have been run and which ones have not/still need to be.
+
+To create a script you can use our custom Rails generator:
```

Comment:
Are there any important rules or things to know/guidelines about creating these scripts? This would be a good place to add those because I feel like the "gotcha"s would be good to know up front.

Reply:
The rules are actually laid out in the generated script which you can see below. Those comments are just for these docs, they are actually put in the class template that is created :)

Reply:
Oh sweet, so when you run the `rails generate data_update ...` generator, you get those commented lines _in_ the class that you generate?

Reply:
Yep!